metpy/calc/basic.py

```diff
@@ -614,6 +615,90 @@ def sigma_to_pressure(sigma, psfc, ptop):
     return sigma * (psfc - ptop) + ptop
 
 
+@exporter.export
+@preprocess_xarray
+def gwfs(scalar_grid, n):
```

Comment:
We need a name for this--replicating GEMPAK functionality is a big 👍 . Replicating naming from the 80's when you had to pay per-character (or so it seems) is something I'd like to not replicate. 😁

How about `gaussian_smooth`?

Also, is `n` the most sensible argument for this function? Personally, I'd like to be able to specify the kernel width in some kind of physical units as well, but we don't have to do this right now. If this is the most sensible interface to you the user, then I'm happy to just roll with it.

Reply:
Yes, `gwfs` is more of a placeholder name to make it clear what GEMPAK function it corresponds to. Looking ahead to eventual implementations of GEMPAK's `sm5s` and `sm9s`, maybe all of these should start with 'smooth', so `smooth_gaussian`, `smooth_5pt`, etc.?

Regarding `n`, it sounds like you are wanting the related GEMPAK function RDFS (Resolution Dependent Filter for Scalar), which takes `dx` instead of `n` as the second argument, and then behind the scenes computes an appropriate `n` based on `dx` and calls GWFS. But I suppose just because they are two separate GEMPAK functions doesn't mean they have to be separate MetPy functions.

Reply:
`smooth_gaussian` sounds good to me.

If both GWFS and RDFS are doing the same calculation, just tweaking the parameter, I’d lean towards putting all of the functionality in `smooth_gaussian`, and just look at the arguments passed in to determine the proper course of action. That doesn’t have to be implemented in this PR, though—I’d be happy just to merge the GWFS equivalent.